.github/actions/bumpVersion/bumpVersion.js

```diff
@@ -1,71 +1,55 @@
-const {exec} = require('child_process');
-const fs = require('fs');
 const core = require('@actions/core');
 const github = require('@actions/github');
-
-// Use Github Actions' default environment variables to get repo information
-// https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables
-const [repoOwner, repoName] = process.env.GITHUB_REPOSITORY.split('/');
+const functions = require('./functions');
```

Comment:
Did you just move these functions to their own module so that they can be imported else where for testing? I'd prefer if the module were given a more descriptive name than just `functions`. Maybe `semverUtils` or `versionUpdater`?

Reply:
I extracted some of the logic because the previous file was doing lots of things (getting the tags, handling race condition, etc. ).

It also made it easy to reuse it somewhere both for the future and for the test.

I am happy to change the name!